src/testing/complete-redux-store-dump.js

```diff
@@ -0,0 +1,2811 @@
+        relations: [],
+        controllerTimestamp: '2019-08-29T19:46:25.985521867Z'
+      },
+      '4b89852b-0633-4fae-8ba2-acf0261d4a03': {
+        model: {
+          name: 'test2',
+          type: 'iaas',
+          cloudTag: 'cloud-google',
+          region: 'us-east1',
+          version: '2.6.6',
+          availableVersion: '',
+          modelStatus: {
+            status: 'available',
+            info: '',
+            data: {},
+            since: '2019-08-26T16:32:59.869183649Z',
+            kind: '',
+            version: '',
+            life: ''
+          },
+          meterStatus: {
+            color: '',
+            message: ''
+          },
+          sla: 'unsupported'
+        },
+        machines: {},
+        applications: {},
+        remoteApplications: {},
+        offers: {},
+        relations: [],
+        controllerTimestamp: '2019-08-29T19:46:26.31534672Z'
+      },
+      '81887357-1c78-4ad6-8b89-4fb01ebec720': {
+        model: {
+          name: 'test3',
+          type: 'iaas',
+          cloudTag: 'cloud-google',
+          region: 'us-east1',
+          version: '2.6.6',
+          availableVersion: '',
+          modelStatus: {
+            status: 'available',
+            info: '',
+            data: {},
+            since: '2019-08-26T16:33:07.803688819Z',
+            kind: '',
+            version: '',
+            life: ''
+          },
+          meterStatus: {
+            color: '',
+            message: ''
+          },
+          sla: 'unsupported'
+        },
+        machines: {},
+        applications: {},
+        remoteApplications: {},
+        offers: {},
+        relations: [],
+        controllerTimestamp: '2019-08-29T19:46:27.123149166Z'
+      },
+      '06b11eb5-abd6-48f3-8910-b54cf5905e60': {
+        model: {
+          name: 'juju-all-the-things',
+          type: 'iaas',
+          cloudTag: 'cloud-google',
+          region: 'us-east1',
+          version: '2.5.4',
+          availableVersion: '',
+          modelStatus: {
+            status: 'available',
+            info: '',
+            data: {},
+            since: '2019-08-20T03:58:06.759061182Z',
+            kind: '',
+            version: '',
+            life: ''
+          },
+          meterStatus: {
+            color: '',
+            message: ''
+          },
+          sla: 'unsupported'
+        },
+        machines: {
+          '1': {
+            agentStatus: {
+              status: 'started',
+              info: '',
+              data: {},
+              since: '2019-08-20T03:58:05.281591141Z',
+              kind: '',
+              version: '2.5.4',
+              life: ''
+            },
+            instanceStatus: {
+              status: 'running',
+              info: 'RUNNING',
+              data: {},
+              since: '2017-03-09T20:40:24.303597554Z',
+              kind: '',
+              version: '',
+              life: ''
+            },
+            dnsName: '104.196.135.3',
```

Comment:
104.196.135.3 happens to be a publicly visible Grafana :)

Reply:
Hah, thanks!